src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -82,6 +82,7 @@ abstract class Kernel implements KernelInterface, RebootableInterface, Terminabl
 
     const END_OF_MAINTENANCE = '11/2022';
     const END_OF_LIFE = '11/2023';
+    const LONG_TERM_SUPPORT = true;
```

Comment:
this needs some automating during the release process as well IIUC?

Reply:
you mean internaly for sf core maintainers?

Reply:
To avoid having a new constant, you can use `MINOR_VERSION` and check that the value is `4`.

Reply:
Yes indeed, I will change it
(If it is ok to omit 2.8 for its last months ;) )

Reply:
2.8 is out of scope as this is a new feature, the flag will be available to 4.4+ applications. 
We may need to update the check in a future major version in case its LTS version is not the `x.4` one